backend/package.json

```diff
@@ -18,6 +18,6 @@
   },
   "devDependencies": {
     "nodemon": "^3.1.3",
-    "prisma": "^5.16.2"
+    "prisma": "^5.15.1"
```

Comment:
Try undoing this. 

Delete the node_modules by running `rm -rf node_modules` and `rm package-lock.json` in your terminal to remove these files, and then run `npm i` to reinstall all of your packages again.